.vscode/tasks.json

```diff
@@ -7,10 +7,12 @@
             "args": [
                 "-fdiagnostics-color=always",
                 "-g",
-                "${workspaceFolder}\\*.cpp",
-                "${workspaceFolder}\\creature\\*.cpp",
-                "${workspaceFolder}\\creature\\main_character\\*.cpp",
-                "${workspaceFolder}\\start_menu\\*.cpp",
+              
+                "${workspaceFolder}\\scr\\*.cpp",
+                "${workspaceFolder}\\scr\\creature\\*.cpp",
+                "${workspaceFolder}\\scr\\utils\\*.cpp",
+                "${workspaceFolder}\\scr\\creature\\main_character\\*.cpp",
```

Comment:
Please keep the alphabetical order of these. See how it was done before.

Reply:
done